packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -294,6 +294,12 @@ export interface ContainerDefinitionOptions {
    * @default - No ports are mapped.
    */
   readonly portMappings?: PortMapping[];
+
+  /**
+   * The type and amount of a resource to assign to a container.
+   * @default - No resources assigned.
+   */
+  readonly resourceRequirements?: ResourceRequirement[];
```

Comment:
I'm wondering if we should have the new field be something that is more specific to EIA, since [resourceRequirements in CFN](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-resourcerequirement.html) technically encompasses both GPU and EIA. GPU currently is specified through its [own field](https://github.com/aws/aws-cdk/pull/13950/files#diff-3654071c3c543c7a4938c726b62afa9b5ad382c05d4d3adff32f883146ff7148L746), and it may be confusing for this field to exist when we have GPU specified elsewhere.

Reply:
I agree, it might get confusing this way. I will make the required updates.

Reply:
I changed the `resourceRequirements` prop to `inferenceAcceleratorResources` (as `inferenceAccelerators` was already a task definiton prop) and changed the type to be a list of device names (strings).